core/src/test/resources/sql/misc.iq

```diff
@@ -738,24 +738,25 @@ EnumerableCalc(expr#0..1=[{inputs}], expr#2=[1], expr#3=[=($t0, $t2)], proj#0..1
 
 # [HIVE-5873] Semi-join to count sub-query
 # [CALCITE-365] AssertionError while translating query with WITH and correlated sub-query
-with parts (PNum, OrderOnHand)
-     as (select * from (values (3, 6), (10, 1), (8, 0)) as t(PNum, OrderOnHand)),
-  supply (PNum, Qty)
```

Comment:
Why you delete this sql test case?

Reply:
There was a issue that prevented the test from passing. Details https://issues.apache.org/jira/browse/CALCITE-5743